crates/bevy_mikktspace/.gitignore

```diff
@@ -0,0 +1,3 @@
+/target/
```

Comment:
I think we can/should remove this gitignore in favor of the current root gitignore

Reply:
Removed.